test/integration/event-handler-test.ts

```diff
@@ -135,7 +135,7 @@ test("options.transform", (done) => {
     },
   });
 
-  eventHandler.on("push", (event: EmitterWebhookEvent) => {
+  eventHandler.on("push", (event: string) => {
```

Comment:
why this change?

Reply:
It's because [the `transform` method is transforming the type into `string`](https://github.com/octokit/webhooks.js/pull/464/files#diff-19ed883872f2faee0e85fe413f91e9718bfa273b0309cc7be942453f6eec30a6R132-R135) (technically, it's type is `"funky"`).

Reply:
Ah of course, yes, thank you!